bot.py

```diff
@@ -267,6 +272,115 @@ async def verif_word_train(message):
     async def ayo(ctx):
         await ctx.send("ayo")
 
+    @bot.command(name = "verify")
+    async def verify(ctx):
+        if not await isWelcome(ctx.message.author) and not await isMod(ctx.guild, ctx.message.author.id):
+            return
+        reference = ctx.message.reference
+        if reference is None:
+            return
+
+        db = [
+            ('🇪🇺', "Europe", []),
+            ('🇦🇱', "Albania", ["Albanian"]),
+            ('🇦🇲', "Armenia", ["Armenian"]),
+            ('🇦🇩', "Andorra", ["Catalan", "Spanish", "French"]),
+            ('🇦🇹', "Austria", ["German"]),
+            ('🇦🇿', "Azerbeijan", ["Azerbaijani"]),
+            ('🇧🇾', "Belarus", ["Belarusian", "Russian"]),
+            ('🇧🇪', "Belgium", ["Dutch", "French", "German"]),
```

Comment:
As discussed yesterday evening, it sounds more convenient not to assume the language a person speaks when they are from Belgium, Switzerland, etc. and let the mod mention the language roles in the command.

Reply:
(Having in mind multilingual countries and adding languages as opt-in sounds more convenient than having in mind the languages of the country that the user doesn't speak)